app/editor/components/EmbeddedMediaLinkEditor.tsx

```diff
@@ -20,54 +16,43 @@ type Props = {
   dictionary: Dictionary;
 };
 
-export function EmbedLinkEditor({ node, view, dictionary }: Props) {
-  const { t } = useTranslation();
-  const embeds = useEmbeds();
-
-  const url = node.attrs.href as string;
+export function EmbeddedMediaLinkEditor({ node, view, dictionary }: Props) {
+  const url = (node.attrs.href ?? node.attrs.src) as string;
   const [localUrl, setLocalUrl] = useState(url);
 
   const moveSelectionToEnd = useCallback(() => {
     const { state, dispatch } = view;
-
     const nextSelection = Selection.findFrom(
       state.tr.doc.resolve(state.selection.from),
       1,
       true
     );
-    const selection = nextSelection ?? TextSelection.create(state.tr.doc, 0);
 
+    const selection = nextSelection ?? TextSelection.create(state.tr.doc, 0);
     dispatch(state.tr.setSelection(selection));
     view.focus();
   }, [view]);
 
-  const openEmbed = useCallback(() => {
+  const openLink = useCallback(() => {
```

Comment:
I'm not seeing open link or remove work correctly on the link editor, are you? 

Honestly they aren't really needed in that context though.

Reply:
yeah, didn't really think it through before, but can just have the input box when for the image.